src/modules/map/components/openlayers/OpenLayersMouseTracker.vue

```diff
@@ -84,6 +84,10 @@ function setDisplayedFormatWithId() {
     text-align: left;
     white-space: nowrap;
 }
+// Show selected area when shift click + drag on map
+:global(.ol-dragzoom) {
+    border: 2px solid red;
```

Comment:
very "nitpicky" but could you use the SASS variable for red, so that we may change it later?
Either "$red" or "$primary" are fine (both can be imported by adding `@import 'src/scss/webmapviewer-bootstrap-theme';` at the top of the `<style>` section, if not there already)